src/rws_client.cpp

```diff
@@ -490,6 +490,10 @@ RWSClient::RWSResult RWSClient::endSubscription()
   return result;
 }
 
+void RWSClient::closeSubscription() {
```

Comment:
```suggestion
void RWSClient::forceCloseSubscription()
{
```
Name suggestion as previous, and curly bracket on new line to be consistent with the rest of the code base.